content/en/docs/Writing policies/match-exclude.md

```diff
@@ -41,6 +41,19 @@ These can be distinguished as:
 * `networking.k8s.io/v1/NetworkPolicy`
 * `crd.antrea.io/v1alpha1/NetworkPolicy`
 
+Wildcards supported formats:
+
+* `Group/*/Kind`
+* `*/Kind`
+* `*`
+
+{{% alert title="Note" color="info" %}}
+* wildcard policy not allowed in background mode.
+* wildcard policy does not support `Generate`, `Verify Images`, `ForEach` rule type.
+* In validate rule type, policy can only deal with Deny and the metadata field of the resource Pattern and AnyPattern.
```

Comment:
"For the `validate` rule type, a policy can only deal with `deny` statements and the `metadata` object in either `pattern` or `anyPattern`. blocks."